app/models/academic_year.rb

```diff
@@ -0,0 +1,13 @@
+# == Schema Information
+#
+# Table name: academic_years
+#
+#  id         :uuid             not null, primary key
+#  starts_on  :date
+#  ends_on    :date
+#  name       :string
```

Comment:
No validation for these fields being present. Or for `ends_on` being greater than `starts_on`.

Personal preference (feel free to ignore): `start_at` or `start_date` over `starts_on`

Reply:
I've adopted a convention over time of `_at` being a `timestamp` attribute and `_on` being a `date` attribute. This is a common convention, e.g. https://github.com/thoughtbot/guides/tree/main/rails